### PR TITLE
Adding contribution guidelines (#162)

### PR DESCRIPTION
**Pull request by kalxas**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+Guide](https://contribution-guide-org.readthedocs.io/).
+
+pygeoapi has the following modes of contribution:
+
+- GitHub Commit Access
+- GitHub Pull Requests
+
+## Code of Conduct
+
+Contributors to this project are expected to act respectfully toward others in accordance with the [OSGeo Code of Conduct](http://www.osgeo.org/code_of_conduct).
```
Comment (tomkralidis) [inline CONTRIBUTING.md]: Should OSGeo link be https?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+## Submitting bugs
+
+### Due diligence
```
Comment (tomkralidis) [inline CONTRIBUTING.md]: Due Diligence ?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+* Proposals to provide developers with GitHub commit access shall be emailed to the pygeoapi [mailing list](https://lists.osgeo.org/mailman/listinfo/pygeoapi). Proposals shall be approved by the pygeoapi development team.  Committers shall be added by the project admin.
+* Removal of commit access shall be handled in the same manner.
+
+### GitHub Pull Requests
+
+* Pull requests may include copyright in the source code header by the contributor if the contribution is significant or the contributor wants to claim copyright on their contribution.
+* All contributors shall be listed at https://github.com/geopython/pygeoapi/graphs/contributors
+* Unclaimed copyright, by default, is assigned to the main copyright holders as specified in https://github.com/geopython/pygeoapi/blob/master/LICENSE
+
+### Version control branching
```
Comment (tomkralidis) [inline CONTRIBUTING.md]: Version Control Branching ?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+      branches. Unless you are sure that your issue affects a non-default
+      branch, __base your branch off the ``master`` one__.
+
+* Note that depending on how long it takes for the dev team to merge your
+  patch, the copy of ``master`` you worked off of may get out of date! 
+    * If you find yourself 'bumping' a pull request that's been sidelined for a
+      while, __make sure you rebase or merge to latest ``master``__ to ensure a
+        speedier resolution.
+
+### Code formatting
```
Comment (tomkralidis) [inline CONTRIBUTING.md]: Code Formatting ?

File: CONTRIBUTING.md (hunk 1 above)
Comment (kalxas) [reply CONTRIBUTING.md]: Fixed, thanks

File: CONTRIBUTING.md (hunk 2 above)
Comment (kalxas) [reply CONTRIBUTING.md]: Fixed, thanks

File: CONTRIBUTING.md (hunk 3 above)
Comment (kalxas) [reply CONTRIBUTING.md]: Fixed, thanks

File: CONTRIBUTING.md (hunk 4 above)
Comment (kalxas) [reply CONTRIBUTING.md]: Fixed, thanks

**Pull request merged**